ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json

```diff
@@ -1769,5 +1769,26 @@
     "CaseFieldID": "judicialMessagesTabHeading",
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "epoExclusionRequirementType",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRUD"
```

Comment:
We should be able to remove the D permissions on this since we will not be removing this at all

Reply:
sorted